### PR TITLE
Update family-tree-builder to 8.2.0.8521 (Support of Catalina)

### PR DESCRIPTION
**Pull request by Najihel**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by vitorgalvao** [comment]
If it works in Catalina now, it’s likely it works on Big Sur as well.

**Comment by Najihel** [comment]
> If it works in Catalina now, it’s likely it works in Big Sur as well.

@vitorgalvao Maybe, I don't know about BigSur and I don't have a BigSur available to verify it.

The official announce is for Catalina and High Sierra only : https://blog.myheritage.com/2020/10/family-tree-builder-now-available-for-macos-catalina-and-macos-high-sierra/

File: Casks/family-tree-builder.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  sha256 "babd0029c007932e0339a97e080608a06ffbc64f77c49879a2b8fd0a93f956c5"
+  version "8.2.0.8521"
+  sha256 "37d553af39747800a9f8b2bfd745ffae2ab521a5894364e377a8be496f36063e"
 
   # mhcache-myheritage.netdna-ssl.com/FP/FamilyTreeBuilder/ was verified as official when first introduced to the cask
   url "https://mhcache-myheritage.netdna-ssl.com/FP/FamilyTreeBuilder/family_tree_builder_#{version.split(".").last}.dmg"
-  appcast "https://www.myheritage.com/FP/FamilyTreeBuilder/appcast.xml"
-  name "Family Tree Builder"
+  name "MyHeritage Family Tree Builder"
+  desc "MyHeritage Family Tree Builder is a free family tree product that allows you to easily create your family tree. Family Tree Builder by MyHeritage supports 36 languages and is absolutely free. It features Smart Matching (a fun way to discover new relatives) and provides excellent support for photos, including face recognition technology."
```
Comment (vitorgalvao) [inline Casks/family-tree-builder.rb:8]: That’s too long and full of marketing fluff and repetition. Leave as it was; it’s descriptive enough.

**Comment by vitorgalvao** [comment]
Is there a way to download this without making an account?

**Comment by Najihel** [comment]
> Is there a way to download this without making an account?

Right here https://www.myheritage.com/page/family-tree-software

**Review by vitorgalvao — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @Najihel.

**Review by miccal — APPROVED** [review]

**Pull request merged**